packages/w3up-client/src/client.js

```diff
@@ -90,6 +90,7 @@ export class Client extends Base {
    * @param {Account.EmailAddress} email
    * @param {object} [options]
    * @param {AbortSignal} [options.signal]
+   * @param {import('@ipld/dag-ucan').Fact[]} [options.facts]
```

Comment:
I'd consider exposing this in the client as `options.appName` or something - the specific "fact" that you want to be passed to the access request. I don't think we should encourage sending arbitrary data here.

Reply:
sure, happy to do it this way! will make the change and get this merged once I do a little more testing